src/status_im2/contexts/share/view.cljs

```diff
@@ -0,0 +1,150 @@
+(ns status-im2.contexts.share.view
+  (:require [utils.i18n :as i18n]
+            [quo2.core :as quo]
+            [react-native.core :as rn]
+            [react-native.safe-area :as safe-area]
+            [status-im2.contexts.share.style :as style]
+            [utils.re-frame :as rf]
+            [reagent.core :as reagent]
+            [quo2.foundations.colors :as colors]
+            [status-im2.common.qr-code.qr :as qr]
+            [clojure.string :as string]
+            [status-im2.constants :as const]
+            [react-native.blur :as blur]
+            ;;TODO(siddarthkay) : move the components below over to status-im2 ns
+            ;; issue -> https://github.com/status-im/status-mobile/issues/15549
```

Comment:
The issue already exists, so I think we can clear up the comments in this ns macro.

Reply:
fixed here : https://github.com/status-im/status-mobile/pull/15528/commits/cc2d1e71421060bba88155d6796b7eb10cf5c9e9